src/test/java/com/wikia/webdriver/common/contentpatterns/MercurySubpages.java

```diff
@@ -29,6 +29,11 @@
 
   // Articles prepared for mercurycc.wikia.com wiki (with FC, CC, TA and TV)
   public static final String CC_MAIN_PAGE = "Mercury_CC_Wikia";
+  public static final String CC_REDIRECT_SOURCE_1 = "RedirectSource1";
+  public static final String CC_REDIRECT_SOURCE_2 = "RedirectSource2";
+  public static final String CC_REDIRECT_DESTINATION_1 = "RedirectDestination1";
+  // it is expected that redirect destination 2 article doesn't exist
+  public static final String CC_REDIRECT_DESTINATION_2 = "RedirectDestination2";
```

Comment:
could you name this variable CC_NOT_EXISTING_REDIRECT_DESTINATION
this way variable would be self explanatory and comment would be redundant?